packages/style-kit/src/_tags.scss

```
@@ -0,0 +1,41 @@
+@import './variables';
+
+@mixin tag-base($theme) {
+  @include tag-label;
+  display: inline-block;
+  padding: 6px 16px 6px 16px;
+  border-radius: $border-radius-large;
+  background: transparentize(map-get($theme, 'highlight'), 0.9);
+  color: map-get($theme, 'highlightText');
+  text-align: center;
+}
+
+@mixin tag-edit {
+  meeco-icon,
+  svg {
+    vertical-align: middle;
+    display: inline-block;
+    padding-left: 0.5rem;
+  }
+}
+
+@mixin tag-icon {
+  meeco-icon,
+  svg {
+    vertical-align: middle;
+    display: inline-block;
+    padding-right: 0.5rem;
+  }
+}
+
+@mixin tags-main($theme) {
+  .tag {
+    @include tag-base($theme);
+    .tag-edit {
+      @include tag-edit;
+    }
+    .tag-icon {
```

Oops - just meant the mixins sorry. The goal is to have html like this:

```
<span class="tag">My Tag</span><span class="tag edit">My Other Tag</span>
```

```suggestion
    &.edit {
      @include tag-edit;
    }
    &.icon {
      @include tag-icon;
    }
```